.changeset/shaggy-garlics-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Support special characters in query paths sent to `getCountFromServer(...)`, `getCount(...)` (lite API), and `getDocs(...)` (lite API). 
```

Comment:
No, these other operations are not affected in my testing. The difference is where the path is included in the request. If the path is specified as part of the endpoint URL, then the special characters need to be escaped by the SDK. Otherwise, if the path is in the proto, no escaping is required.